XcodeServerSDK/XcodeServerConfig.swift

```diff
@@ -33,16 +42,51 @@ public class XcodeServerConfig : JSONSerializable {
         return dict
     }
     
-    public init(var host: String, user: String?, password: String?) {
+    ///
+    /// Initializes a server configuration with the provided host.
```

Comment:
I'm not convinced, @esttorhe do you reckon the different parsing is a bug or a desired behavior? I'd still prefer to keep it javadoc-style.